services/motion/builtin/builtin_test.go

```diff
@@ -406,7 +408,11 @@ func TestObstacleReplanningSlam(t *testing.T) {
 		ExecutionID:   executionID,
 		LastPlanOnly:  true,
 	})
-	test.That(t, err, test.ShouldBeNil)
+	if err != nil {
```

Comment:
I've updated this test to be little more robust. We were landing cases where sometimes the error was nil and sometimes it was `"no need to move, already within planDeviationMM: plan failed". Both of which are success cases imo.